### PR TITLE
dist: build linux/darwin arm64 targets

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
Adding the linux/darwin arm64 targets, we start building arm64 for the server and will start adding the targets for the plugins for the ones we prepackage in the build

related to: https://github.com/mattermost/mattermost-server/pull/18491

#### Ticket Link
related to https://mattermost.atlassian.net/browse/DOPS-633

### CODE REVIEW

**Comment by cpanato** [comment]
@hanzei made the requested changes PTAL

**Review by hanzei — APPROVED** [review]

File: server/telemetry.go [hunk 1]
```diff
@@ -28,24 +27,23 @@ func (p *Plugin) initTracker() {
...
 		}
 	}
-	logger := logger.NewLogger(logger.Config{}, p.API, nil, "")
-	p.tracker = telemetry.NewTracker(p.client, p.API.GetDiagnosticId(), p.API.GetServerVersion(), manifest.Id, manifest.Version, "nps", enableDiagnostics, logger)
+	p.tracker = telemetry.NewTracker(p.client, p.API.GetDiagnosticId(), p.API.GetServerVersion(), manifest.ID, manifest.Version, "nps", enableDiagnostics)
 }
 
 func (p *Plugin) sendScore(score int, userID string, timestamp int64) {
-	p.tracker.TrackUserEvent(NpsScore, userID, p.getEventProperties(userID, timestamp, map[string]interface{}{
+	p.tracker.TrackUserEvent(NpsScore, userID, p.getEventProperties(userID, timestamp, map[string]interface{}{ // nolint: errcheck
```
Comment (hanzei) [inline server/telemetry.go:34]:
```suggestion
	_ = p.tracker.TrackUserEvent(NpsScore, userID, p.getEventProperties(userID, timestamp, map[string]interface{}{
```

Comment (hanzei) [reply server/telemetry.go:34]: Same suggestion for the two changes below

Comment (cpanato) [reply server/telemetry.go]: done, thanks

**Review by hanzei — APPROVED** [review]

**Pull request merged**